lms/djangoapps/philu_overrides/views.py

```diff
@@ -519,17 +519,7 @@ def course_about(request, course_id):
     from commerce.utils import EcommerceService
     from course_modes.models import CourseMode, get_cosmetic_display_price
     from lms.djangoapps.courseware.access_utils import ACCESS_DENIED
-    from lms.djangoapps.courseware.views.views import registered_for_course
```

Comment:
Good work on cleaning this, please verify course about page thoroughly. You can resolve this if you've.

Reply:
get_courses, sort_by_start_date, get_course_by_id, sort_by_announcement are already imported on top of the file. 
get_permission_for_course_about, get_course_overview_with_access, get_course_with_access are unused.

That's why it was safe to remove all this extra imports, I also test course_about page.